### ISSUE TITLE
Folder renaming not possible

### ISSUE THREAD

**Issue by ToruiDev** [afterlogic/aurora-files#11]
I installed from package yesterday.
I went and setup the Database which said "Tables created/updated successfully."

The Following Tables were created
```
SHOW TABLES;
+-------------------------------+
| Tables_in_torui_aurora        |
+-------------------------------+
| au_adav_addressbookchanges    |
| au_adav_addressbooks          |
| au_adav_cache                 |
| au_adav_calendarchanges       |
| au_adav_calendarinstances     |
| au_adav_calendarobjects       |
| au_adav_calendars             |
| au_adav_calendarsubscriptions |
| au_adav_cards                 |
| au_adav_groupmembers          |
| au_adav_locks                 |
| au_adav_propertystorage       |
| au_adav_reminders             |
| au_adav_schedulingobjects     |
| au_eav_attributes_bigint      |
| au_eav_attributes_bool        |
| au_eav_attributes_datetime    |
| au_eav_attributes_double      |
| au_eav_attributes_int         |
| au_eav_attributes_mediumblob  |
| au_eav_attributes_string      |
| au_eav_attributes_text        |
| au_eav_entities               |
| au_min_hashes                 |
+-------------------------------+
24 rows in set (0.001 sec)
```

When trying to rename a folder in either Corporate or private, it fails with the following error message
!Dateiumbenennung fehlgeschlagen. (SQLSTATE[42S02]: Base table or view not found: 1146 Table 'torui_aurora.au_adav_sharedfiles' doesn't exist)" (German language) which translates to "could not rename file". Repeating the step of logging in as superadmin, Database Settings, Create/Update Tables results in the same message of "Tables created/updated successfully." and no change to the renaming behaivor.

Using php 7.1 and Apache on CentOS

**Comment by afterlogic-support** [comment]
I've asked the developers to look into this. Will post an update once there's a response from them. Thanks.

**Comment by afterlogic-support** [comment]
We've just released version 8.2.17 of Aurora Files, the issue is corrected there.
